src/test/java/teammates/test/cases/util/FieldValidatorTest.java

```diff
@@ -367,9 +368,15 @@ public void invalidityInfoFor_validGender_returnEmptyString() {
     public void invalidityInfoFor_invalidGender_returnErrorString() {
```

Comment:
But the nationality field is not updated in that line, it just passes the sanitized version of `nationality` to String.format without updating `nationality` of the `StudentProfileAttributes` object.